harper-core/tests/text/linters/The Great Gatsby.snap.yml

```diff
@@ -219,6 +219,15 @@ Suggest:
 
 
 
+Lint:    Punctuation (31 priority)
+Message: |
+      61 | restless. Instead of being the warm centre of the world, the Middle West now
+         |           ^~~~~~~ Discourse markers at the beginning of a sentence should be followed by a comma.
```

Comment:
I'm much more confident that this one is a false positive. I'd assume the `of` after the `instead` makes the difference.